galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.manager/src/main/java/dev/galasa/cicsts/internal/CicstsManagerImpl.java

```diff
@@ -206,6 +206,7 @@ public ICicsTerminal generateCicsTerminal(Field field, List<Annotation> annotati
         CicsTerminal annotation = field.getAnnotation(CicsTerminal.class);
 
         String tag = defaultString(annotation.cicsTag(), "PRIMARY").toUpperCase();
+        String loginCredentials = defaultString(annotation.loginCredentials(), "").toUpperCase();
         

```

Comment:
if this fails to find any logon credentials we should go to the zos image and get its default credentials

Reply:
@hobbit1983 If the credentials are blank, that bypasses the new CESL behaviour leaving the code to do what it currently does.
If we default it to use the default credentials, it means the CESL code will always be ran. This would then be new behaviour for all users. Would that potentially break somebody? (I don't know if logging in via CESL using the default creds makes a difference or not?)

Reply:
Agreed with @hobbit1983 & Colin in a meeting that if `loginCredentialsTag` isn't provided, CESL won't be attempted.